components/ChipTextField/examples/MDCChipTextField.m

```diff
@@ -13,14 +13,22 @@
 // limitations under the License.
 
 #import "MDCChipTextField.h"
+
 #import "MaterialChips.h"
+#import "MDCChipTextFieldScrollView.h"
+
+//static CGFloat const kChipsSpacing = 10.0f;
```

Comment:
Should we remove this?

Reply:
Addressed in efcb81c4eed97ffe93b9378271fa5c91d227a63a. 

I added the support for the spacing between chips back, but set the value to be 0 for now, since removing this influence factor would be easier for us to debug before all other alignment issues are fixed.